wallet/db.c

```diff
@@ -2165,6 +2165,323 @@ u8 *db_column_talarr(const tal_t *ctx, struct db_stmt *stmt, int col)
 			   db_column_bytes(stmt, col), 0);
 }
 
+/* Modern variants: by name */
+u64 db_col_u64(struct db_stmt *stmt, const char *colname)
+{
+	size_t col = db_query_colnum(stmt, colname);
+
+	if (db_column_is_null(stmt, col)) {
```

Comment:
Why is this more complex than `return db_column_u64(stmt, db_query_colnum(stmt, colname));`?  The existing `db_column_`* functions do the same checks.  Ditto for all the other `db_col_`* functions.

Reply:
I cut and paste: I'll be removing the old functions

Reply:
Done!